From 40f9ce185c0582e3ba2b5f1381d82b6558a70916 Mon Sep 17 00:00:00 2001
From: Michal Hrusecky <michal@hrusecky.net>
Date: Tue, 9 Apr 2024 21:53:57 +0200
Subject: [PATCH] hwdata: Add Marvell to pci.ids

Add Marvell SDIO Wi-Fi card vendor to the pci.ids
---
 utils/hwdata/Makefile                       |  2 +-
 utils/hwdata/patches/add-marvell-sdio.patch | 12 ++++++++++++
 2 files changed, 13 insertions(+), 1 deletion(-)
 create mode 100644 utils/hwdata/patches/add-marvell-sdio.patch

diff --git a/utils/hwdata/Makefile b/utils/hwdata/Makefile
index 0ced19e01..5dcc12959 100644
--- a/utils/hwdata/Makefile
+++ b/utils/hwdata/Makefile
@@ -7,7 +7,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=hwdata
 PKG_VERSION:=0.359
-PKG_RELEASE:=1
+PKG_RELEASE:=2
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://codeload.github.com/vcrhonek/hwdata/tar.gz/v$(PKG_VERSION)?
diff --git a/utils/hwdata/patches/add-marvell-sdio.patch b/utils/hwdata/patches/add-marvell-sdio.patch
new file mode 100644
index 000000000..5aafdd4ec
--- /dev/null
+++ b/utils/hwdata/patches/add-marvell-sdio.patch
@@ -0,0 +1,12 @@
+Index: hwdata-0.359/pci.ids
+===================================================================
+--- hwdata-0.359.orig/pci.ids
++++ hwdata-0.359/pci.ids
+@@ -80,6 +80,7 @@
+ # SpeedStream is Efficient Networks, Inc, a Siemens Company
+ 02ac  SpeedStream
+ 	1012  1012 PCMCIA 10/100 Ethernet Card [RTL81xx]
++02df  Marvell
+ 02e0  XFX Pine Group Inc. (Wrong ID)
+ 0303  Hewlett-Packard Company (Wrong ID)
+ 0308  ZyXEL Communications Corporation (Wrong ID)
-- 
2.44.0

